src/test/java/com/spotify/docker/PushMojoTest.java

```diff
@@ -47,4 +47,14 @@ public void testPush() throws Exception {
     verify(docker).push(eq("busybox"), any(AnsiProgressHandler.class));
   }
 
+  public void testPushPrivateRepo() throws Exception {
```

Comment:
I'm not sure how to get this test to use `src/test/resources/private-repo-settings.xml`. It needs that to pass.